ext/http/http_next.rs

```diff
@@ -1034,6 +1035,34 @@ impl UpgradeStream {
     .try_or_cancel(cancel_handle)
     .await
   }
+
+  async fn write_vectored(
+    self: Rc<Self>,
+    buf1: &[u8],
```

Comment:
**question**: Is there a particular reason for supporting specifically two buffers worth of data?

Reply:
Rust's `write_all_vectored` is not stable, this function is a stable version of that but for 2 buffers. We can support variable size once its stable.